src/app/item-popup/PlugTooltip.m.scss

```diff
@@ -59,7 +59,7 @@
     @include tooltip-section-color($red);
   }
   .automaticallyPickedSection {
-    @include tooltip-section-color($orange);
+    @include tooltip-section-color(#e8a534);
```

Comment:
Meant to use a variable?

Reply:
Unfortunately an instance where I had to hard code the colour to avoid creating a dedicated sass variable for this. Unless i'm mistaken (and i'll double check) mixins don't support css vars as parameters. 

Since all the other tooltip background tints are also predefined colour, this feels like an acceptable place to have a hard coded colour reference. Not sure if you agree?